app/scripts/controllers/permissions/index.js

```diff
@@ -510,6 +512,75 @@ export class PermissionsController {
     })
   }
 
+  /**
+   * Removes all known domains and their related permissions.
+   */
+  clearPermissions () {
```

Comment:
This function was moved, because its placement didn't make any sense. It hasn't been changed.